### PR TITLE
Path limit

### PR DESCRIPTION
**Pull request by daschuer**
This should reduce the name length of the resulting zip

### CODE REVIEW

**Comment by JoergAtGithub** [comment]
The shorter path solves my problems with unzipping the buildenv file!
Thank you!

PS: It's still a zip in a zip, but I guess this is because GitHub zips every artifact again.

**Comment by daschuer** [comment]
Than you for testing. 

The zip in zip issue is a known issue without a solution yet:
https://github.com/actions/upload-artifact/issues/39

@Holzhaus merge?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -61,15 +61,21 @@ jobs:
         wavpack
       VCPKG_DEFAULT_TRIPLET: ${{ matrix.vcpkg_triplet }}
       VCPKG_OVERLAY_PORTS: overlay/ports
+      DEPS_BASE_NAME: mixxx-deps
```
Comment (Holzhaus) [inline .github/workflows/build.yml:64]: We could go back to the even shorter `buildenv` name. :D

Comment (daschuer) [reply .github/workflows/build.yml:64]: I don't really mind. To put Mixxx in the name may remind drive by contributors what the file is for when finding it in the Downloads folder. 

Please propose the final name and let's merge.

**Pull request merged**